src/pages/ahnhoongi/Login/Login.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import './Login.scss';
+import { useNavigate } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+const LoginHG = () => {
+  const navigate = useNavigate();
+
+  // function goToMain() {
+  //   fetch('http://10.58.4.56:8000/users/signup', {
+  //     method: 'POST',
+  //     body: JSON.stringify({
+  //       email: idInput,
+  //       password: pwInput,
+  //     }),
+  //   })
+  //     .then(response => response.json())
+  //     .then(result => console.log('결과: ', result));
+  //   // navigate('/main-ahnhoongi');
+  // }
+
+  function goToMain() {
+    // fetch('http://10.58.4.56:8000/users/signin', {
+    //   method: 'POST',
+    //   body: JSON.stringify({
+    //     email: idInput,
+    //     password: pwInput,
+    //   }),
+    // })
+    //   .then(response => response.json())
+    //   .then(result => console.log('결과: ', result));
+    navigate('/main-ahnhoongi');
+  }
+
+  const [idInput, setIdInput] = useState('');
+
+  const handleIdInput = event => {
+    setIdInput(event.target.value);
+  };
+
+  const [pwInput, setPwInput] = useState('');
+
+  const handlePwInput = event => {
+    setPwInput(event.target.value);
+  };
+
+  const [isActive, setIsActive] = useState(false);
+
+  const active = () => {
+    idInput.includes('@') && pwInput.length >= 6
+      ? setIsActive(true)
+      : setIsActive(false);
+  };
+
+  const enter = event => {
+    if (isActive) {
+      if (event.key === 'Enter') {
+        goToMain();
+      }
+    }
+  };
+
+  return (
+    <main className="login-page">
+      <div className="logo">
+        <Link to="">Westagram</Link>
+      </div>
```

Comment:
```suggestion
        <Link to="" className="logo">Westagram</Link>
```

div로 Link 태그를 감싸주지 않아도 됩니다!
불필요한 deps를 늘리지 않도록 주의해주세요!